src/Datasource/EntityTrait.php

```diff
@@ -1231,7 +1231,7 @@ public function source($alias = null)
      */
     public function __toString()
     {
-        return json_encode($this, JSON_PRETTY_PRINT);
+        return json_encode($this, JSON_PRETTY_PRINT | JSON_UNESCAPED_UNICODE);
```

Comment:
@markstory D'oh! Haven't though of that use case...

What do you think about using a protected field `_json_encode_options` defaulting to `JSON_PRETTY_PRINT`? So everone wanting unescaped unicode could set the options accordingly in his/her own entities.